execution/executor-benchmark/src/lib.rs

```diff
@@ -108,6 +108,7 @@ pub fn run_benchmark<V>(
     pruner_config: PrunerConfig,
     enable_storage_sharding: bool,
     pipeline_config: PipelineConfig,
+
```

Comment:
Format checker that can be run locally:
cargo +nightly fmt && cargo xclippy &&  cargo sort --grouped --check --workspace

(this is enforced anyways)